package.json

```diff
@@ -25,5 +28,8 @@
   "engine": {
     "node": ">=0.8.17"
   },
-  "author": "omphalos"
+  "author": "omphalos",
+  "dependencies": {
+    "nodeunit": "^0.9.1"
+  }
```

Comment:
I think we can also change the reference to `npm install -g nodeunit` in the README to `npm install`.